packages/js-client/src/types.ts

```diff
@@ -0,0 +1,7 @@
+import { ContextParams } from '@aragon/sdk-client-common';
+
+export type SimpleStorageContextParams = ContextParams & {
+  // optional so we can set default values for the parameter
+  myParam?: string;
+  // add custom params
+};
```

Comment:
Isn't this redundant with the `context.ts` type definitions?

Reply:
Mmm nope, this is for adding a new property to the context params if we want

Reply:
See `packages/js-client/src/context.ts`. There's the same local definition, but for state and overridden